docker/images/kuma_base/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:2.7-slim
+FROM python:2.7-slim-jessie
```

Comment:
Confirmed, ``python:2.7-slim`` moved from being the same tag as ``python:2.7-slim-jessie`` to ``python:2.7-slim-stretch``:

https://github.com/docker-library/official-images/pull/4385/files#diff-051bb6df61b6c6e7f8c1868985011b07R43